docs/reference/sql/endpoints/jdbc.asciidoc

```diff
@@ -51,7 +51,7 @@ Once registered, the driver understands the following syntax as an URL:
 
 ["source","text",subs="attributes"]
 ----
-jdbc:es://[[http|https]://]*[host[:port]]*/[prefix]*<[?[option=value]&]*
+jdbc:es://[[http|https]://]?[host[:port]]?/[prefix]?[?[option=value]&]*
```

Comment:
`?` means zero ore one in regex, which is what we want, but how can we differentiate from the `[?[option...]` which is the actual char `?` ? In the error message I used `\?`, but not sure. Any suggestions?

Reply:
For 0 or 1 (or `?`) there is also the `{0,1}` _more explicit_ notation, but I am not sure if we want to complicate things...